python/cudf/cudf/core/column/string.py

```diff
@@ -5594,16 +5594,15 @@ def data(self):
         return self._data
 
     def all(self, skipna: bool = True) -> bool:
-        # The skipna argument is only used for numerical columns.
-        # If all entries are null the result is True, including when the column
-        # is empty.
-
+        if self.null_count == self.size:
+            return True
```

Comment:
If `skipna` is `False`, pandas 2.1.4 raises an error. Do we need to check that here?

```python
import pandas as pd
s = pd.Series([pd.NA, pd.NA])
print(s.all(skipna=True))  # True
print(s.all(skipna=False))  # TypeError: boolean value of NA is ambiguous
```

```suggestion
        if skipna and self.null_count == self.size:
            return True
```

Reply:
Thanks for catching this @bdice, fixed it.